### ISSUE TITLE
Uncaught ReferenceError: process is not defined

### ISSUE THREAD

**Issue by arifsupriyadi** [sassy-labs/datepicker#5]
I use react js version 17, it's not working. Uncaught ReferenceError: process is not defined

**Comment by jhonny53** [comment]
+1

**Comment by siddharthborderwala** [comment]
Thanks for bringing this issue to my attention. It is fixed and you can resolve it by upgrading to v0.8.0 of sassy-datepicker

You can run
```sh
yarn add sassy-datepicker@0.8.0
# or
npm install sassy-datepicker@0.8.0
```
